tensorboard/webapp/feature_flag/store/feature_flag_types.ts

```diff
@@ -25,3 +25,12 @@ export interface FeatureFlagState {
 export interface State {
   [FEATURE_FLAG_FEATURE_KEY]?: FeatureFlagState;
 }
+
+export const initialState: FeatureFlagState = {
+  isFeatureFlagsLoaded: false,
+  features: {
+    enabledExperimentalPlugins: [],
+    inColab: false,
+    enableGpuChart: false,
+  },
+};
```

Comment:
It is a bit weird that this is defined in the types file. Can this just move to the default provider?

Reply:
Done.